_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,158 @@
+// ------------------------------
+// edX Pattern Library: Variables
+
+// About: Sass partial for defining variables used throughout the applications.
+
+// #Grid:                    Grid and layout variables
+// #Fonts:                   Font definitions
+// #Colors (Palette):        Base and theme color definitions
+// #Colors (Application):    Colors for use within the entire application
+// #Timing:                  Event timing variables
+// ----------------------------
+
+
+// ----------------------------
+// #Grid
+// ----------------------------
+$baseline:     rem(20);
+
+$columns:      12;
+$min-width:    rem(768);
+$max-width:    rem(1600);
+
+
+// ----------------------------
+// #Fonts
+// ----------------------------
+$font-sans:     'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-serif:    'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
+$font-size:     1rem;
+
+
+// ----------------------------
+// #Colors (Palette)
+// ----------------------------
+// Start configuration...
+// - - - - - - - - - - - - - -
+$transparent:    rgba(0, 0, 0, 0);
+$black:          rgba(0, 0, 0, 1.0);
+$gray:           rgba(127, 127, 127, 1.0);
+$white:          rgba(255, 255, 255, 1.0);
+$shadow:         rgba($black, 0.20);
+$highlight:      rgba($white, 0.30);
+$red:            rgba(178, 6, 16, 1.0);
+$orange:         rgba(237, 189, 60, 1.0);
+$yellow:         rgba(237, 189, 60, 1.0);
+$green:          rgba(37, 184, 90, 1.0);
+$blue:           rgba(0, 159, 230, 1.0);
+$indigo:         rgba(75, 0, 130, 1.0);
+$violet:         rgba(143, 0, 255, 1.0);
+$pink:           rgba(183, 37, 103, 1.0);
+// - - - - - - - - - - - - - -
+// End configuration
+// ----------------------------
+$brand-primary:      $blue;
```

Comment:
Yeah I can do that. I was thinking though, that folks who made a theme would be familiar with their brand colors and know what the primary versus secondary colors were. For example, MIT's style guide clearly states which is which.